### DISCUSSION TITLE
Dashboard site categorisation

### DISCUSSION THREAD

**Discussion by GarveScott-Lodge** [plausible/analytics#1774] [Anything Else]
I currently have 19 domains/websites within my Plausible account which means they are listed across 2 pages in the My Sites area.

I hope this number will increase a lot in the next year or two. It would be really useful if I could tag or categorise these sites with the ability to filter sites so I can just see those in a particular category.

**Comment by metmarkosaric** [comment]
thanks @GarveScott-Lodge! there's a similar thread already so i'll close this one. please post your comment to: https://github.com/plausible/analytics/discussions/286
